bundles/org.openhab.binding.bosesoundtouch/src/main/java/org/openhab/binding/bosesoundtouch/internal/PresetContainer.java

```diff
@@ -41,6 +44,7 @@ public class PresetContainer {
      * Creates a new instance of this class
      */
     public PresetContainer(Storage<ContentItem> storage) {
+        mapOfPresets = new HashMap<>();
```

Comment:
It's even better to assign the value where the field is declared and add the `final` modifier to the field. While you're at it you might also wanto to change the field type from the `HashMap` implementation to the `Map` interface.